assimilation_code/programs/readme.rst

```diff
@@ -0,0 +1,195 @@
+
+Changing Observation Sequence Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`obs_keep_a_few/obs_keep_a_few <obs_keep_a_few/obs_keep_a_few.html>`_
+   Creates an output observation sequence file that is shorter than the input obs_seq file.
+ 
+`obs_selection/obs_selection <obs_selection/obs_selection.html>`_
+   Extracts observations out of one or more obs_sequence files
+   according to a  list of observation types, times, and locations.
+   The list is usually created by :doc:`obs_seq_coverage/obs_seq_coverage`, 
+   but can be an observation sequence file,
+ 
+`obs_sequence_tool/obs_sequence_tool <obs_sequence_tool/obs_sequence_tool.html>`_
+   Subsets, combines, or alters observations from one or more observation sequence files 
+   and optionally writes them into a single output obs_seq file.
+
+`obs_loop/obs_loop <obs_loop/obs_loop.html>`_
+   A template to read in observations from one obs_seq file and write them,
+   optionally modified by user supplied code, to another obs_seq file.
+ 
+obs_utils/obs_sort
+   Do a complete sort of an obs_seq file by location, observation type, then variance.
+   An ancestor of obs_remove_dups.
+
+obs_utils/obs_remove_dups
+   Removes duplicate observations from an obs_seq file, which involves a complete sort
+   by time, location, observation type, then variance.
+ 
+`obs_common_subset/obs_common_subset <obs_common_subset/obs_common_subset.html>`_
+   Select the subset of observations, which were successfully assimilated, 
+   from two or more assimilation cases (which used the same obs_seq.out file).
+ 
+`obs_seq_verify/obs_seq_verify <obs_seq_verify/obs_seq_verify.html>`_
+   Reorders the observations from a forecast run of DART into a structure 
+   that is amenable for the evaluation of the forecast.
+ 
+
+obs_utils/obs_data_denial
+   THIS IS NOT YET DONE!
+   Help implement a data-denial experiment by randomly changing the error variance
+   of N of each obs type in an observation sequence file to a huge value.
+ 
+Assimilation Programs
+-----------------------------------
+ 
+`perfect_model_obs/perfect_model_obs <perfect_model_obs/perfect_model_obs.html>`_
+   Creates synthetic observation sequences from a hindcast model.
+ 
+`filter/filter <filter/filter.html>`_
+   Main Fortran program for driving ensemble filter assimilations.
+
+filter/filter.separate_seq
+   Like filter, but each task updates its own sequence in obs_space_diagnostics.
+   Included here just for future reference.
+
+`advance_time/advance_time <advance_time/advance_time.html>`_
+   Provides a shell-scripting-friendly way to increment and decrement calendar dates and times.
+ 
+`integrate_model/integrate_model <integrate_model/integrate_model.html>`_
+   Generic main program which advances a single ensemble member in ``perfect_model_obs`` 
+   or the serial or parallel version of the ``filter`` program.
+
+`integrate_model/integrate_model_parallel <integrate_model/integrate_model.html>`_
```

Comment:
combine these, or remove the parallel version altoghterh

Reply:
removed parallel version (not used anywhere it seems)